src/desktop/apps/article/routes.js

```diff
@@ -124,12 +126,18 @@ export async function index(req, res, next) {
         "../../../components/main_layout/templates/react_blank_index.jade"
     }
 
-    const { CURRENT_USER, IS_MOBILE, IS_TABLET } = res.locals.sd
+    const {
+      CURRENT_USER,
+      EDITORIAL_COLLECTIONS_RAIL, // TODO: Remove after A/B test
+      IS_MOBILE,
+      IS_TABLET,
+    } = res.locals.sd
 
     const isMobile = IS_MOBILE
     const isTablet = IS_TABLET
     const jsonLD = stringifyJSONForWeb(articleModel.toJSONLD())
     const customEditorial = isCustomEditorial(article.id)
+    const showCollectionsRail = Boolean(EDITORIAL_COLLECTIONS_RAIL)
```

Comment:
I wonder if it's worth combining the logic I have in the `CanvasFooter` to ensure this only displays for standard, feature, and news layouts here.

```
const showCollectionsRail = ["standard", "feature", "news"].includes(article.layout)
```

Reply:
Actually we don't have access to the layout at this point, correct?

Reply:
This is a great idea, and yes, we do have access to the layout -- will add this